test/test_eager_transforms.py

```diff
@@ -720,22 +720,22 @@ def foo(x):
         x = torch.tensor(3.14, device=device)
         functorch.grad(foo)(x)
 
-    @onlyCPU
     def test_tensor_print(self, device):
-        x = torch.tensor(3.14, device=device)
+        x = torch.tensor([[3.14, ], [3.14, ]], device=device)
```

Comment:
Could we keep the original tests and add some new test cases? It's probably valuable to test a variety of tensors to make sure they can be printed:
- torch.tensor(3.14, device=device)
- torch.tensor([[3.14, ], [3.14, ]], device=device)
- Some very big tensor (`torch.randn(64, 3, 32, 32)`). We can probably compare that `repr(t_inside_transform)` is `GradTrackingTensor(lvl=2, value={repr(t)}` or something

Reply:
There are probably other edge cases that we want to test that I'm not thinking of here